sockets3.py

```diff
@@ -0,0 +1,239 @@
+#OBSOLETE, WILL BE REMOVED IN THE FUTURE
```

Comment:
I don't think we need this file anymore, maybe it was a leftover from testing?

Reply:
Yes, sockets3 became obsolete after out last brainstorm, I removed it. 
Also did more cleanup